docs/command-reference/tanzu_apps_workload_apply.md

```diff
@@ -73,12 +73,11 @@ tanzu apps workload apply --file workload.yaml
 ```
       --context name      name of the kubeconfig context to use (default is current-context defined by kubeconfig)
       --kubeconfig file   kubeconfig file (default is $HOME/.kube/config)
-      --no-color          disable color output in terminals
+      --no-color          deactivate color, bold, and emoji output
   -v, --verbose int32     number for the log level verbosity (default 1)
 ```
 
 ### SEE ALSO
 
 * [tanzu apps workload](tanzu_apps_workload.md)	 - Workload lifecycle management
-* [environment variable](../working-with-workloads.md#env-vars) support
```

Comment:
Why is this removed as part of this PR?

Reply:
Maybe it happened when I ran `make docs` or when rebasing from `main`, I'll remove it. Thanks for pointing this out.

Reply:
This is a problem with `make docs` command. Every time I run it, this line is getting erased. And I have to it because of the change to `no-color` explanation.